dynawo/sources/Models/Modelica/Dynawo/Electrical/HVDC/HvdcVSC/BaseControls/DCVoltageControl/BaseDCVoltageControl.mo

```diff
@@ -59,13 +59,15 @@ model BaseDCVoltageControl "Base DC Voltage Control for the HVDC VSC model"
     Placement(visible = true, transformation(origin = {-110, 6}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
   Modelica.Blocks.Math.Gain deltaVDCCalc(k = RdcPu * SNom / SystemBase.SnRef) annotation(
     Placement(visible = true, transformation(origin = {-50, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
-  Modelica.Blocks.Continuous.FirstOrder firstOrder(T = 1e-4, y_start = Ip0Pu) annotation(
-    Placement(visible = true, transformation(origin = {-80, 50}, extent = {{10, -10}, {-10, 10}}, rotation = 0)));
 
   parameter Types.VoltageModulePu Udc0Pu "Start value of dc voltage in pu (base UdcNom)";
   parameter Types.VoltageModulePu UdcRef0Pu "Start value of dc voltage reference in pu (base UdcNom)";
   parameter Types.VoltageModulePu U0Pu "Start value of ac voltage in pu (base UNom)";
   parameter Types.PerUnit Ip0Pu "Start value of active current in pu (base SNom)";
+  Modelica.Blocks.Continuous.FirstOrder firstOrder1(T = 0.01, y_start = U0Pu) annotation(
+    Placement(visible = true, transformation(origin = {-177, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Continuous.FirstOrder firstOrder2(T = 0.01, y_start = Udc0Pu) annotation(
+    Placement(visible = true, transformation(origin = {-140, -80}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
```

Comment:
The two first orders should be put with the other blocks (before the parameter Udc0Pu)